src/odemis/driver/xt_client.py

```diff
@@ -2279,6 +2286,58 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                 # know the user that the move is not complete.
                 if future._must_stop.is_set():
                     raise CancelledError()
+
+                # TODO: extend the behavior if rotation, tilt axes also need to be checked.
+                # So far checking the above is not required, as we have observed reporting stage position only in x/y
+                # being compromised. The issue with case western system was that the stage metadata was not
+                # updated in timely manner during the overview acquisition in FM mode (during which rotation, tilt are
+                # not changed), resulting in misplaced tiles in the image display. Even when the stage has physically
+                # moved to the target position but took time in updating its actual position through metadata
+                # The issue is resolved by checking if only x and y stage metadata reports the target position properly
```

Comment:
As described below, I think we can easily implement it, by loosing a little bit the requirements in the case of rotation. So this TODO could go away.
Also, in general, please refrain from mentioning specific names of installations in public code.